xgcm/grid.py

```diff
@@ -37,106 +40,42 @@ def _maybe_promote_str_to_list(a):
 _VALID_BOUNDARY = [None, "fill", "extend", "extrapolate"]
 
 
+_XGCM_BOUNDARY_KWARG_TO_XARRAY_PAD_KWARG = {
+    "periodic": "wrap",
+    "fill": "constant",
+    "extend": "edge",
+}
+
+
+def default_boundary():
+    return "periodic"
+
+
 class Axis:
-    """
-    An object that represents a group of coordinates that all lie along the same
-    physical dimension but at different positions with respect to a grid cell.
-    There are four possible positions:
-
-         Center
-         |------o-------|------o-------|------o-------|------o-------|
-               [0]            [1]            [2]            [3]
-
-         Left
-         |------o-------|------o-------|------o-------|------o-------|
-        [0]            [1]            [2]            [3]
-
-         Right
-         |------o-------|------o-------|------o-------|------o-------|
-                       [0]            [1]            [2]            [3]
-
-         Inner
-         |------o-------|------o-------|------o-------|------o-------|
-                       [0]            [1]            [2]
-
-         Outer
-         |------o-------|------o-------|------o-------|------o-------|
-        [0]            [1]            [2]            [3]            [4]
-
-    The `center` position is the only one without the `c_grid_axis_shift`
-    attribute, which must be present for the other four. However, the actual
-    value of `c_grid_axis_shift` is ignored for `inner` and `outer`, which are
-    differentiated by their length.
-    """
```

Comment:
Probably want to find a better place to preserve the nice explanation in this docstring.